[2025-09-18T04:29:07.016+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: batch_ingestion_dag.extract_mysql manual__2025-09-18T04:29:04.296968+00:00 [queued]>
[2025-09-18T04:29:07.022+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: batch_ingestion_dag.extract_mysql manual__2025-09-18T04:29:04.296968+00:00 [queued]>
[2025-09-18T04:29:07.023+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 2
[2025-09-18T04:29:07.032+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): extract_mysql> on 2025-09-18 04:29:04.296968+00:00
[2025-09-18T04:29:07.040+0000] {standard_task_runner.py:60} INFO - Started process 213 to run task
[2025-09-18T04:29:07.043+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'batch_ingestion_dag', 'extract_mysql', 'manual__2025-09-18T04:29:04.296968+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/batch_ingestion_dag.py', '--cfg-path', '/tmp/tmpgwl36kkh']
[2025-09-18T04:29:07.048+0000] {standard_task_runner.py:88} INFO - Job 2: Subtask extract_mysql
[2025-09-18T04:29:07.093+0000] {task_command.py:423} INFO - Running <TaskInstance: batch_ingestion_dag.extract_mysql manual__2025-09-18T04:29:04.296968+00:00 [running]> on host b9864b12bd2e
[2025-09-18T04:29:07.156+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='batch_ingestion_dag' AIRFLOW_CTX_TASK_ID='extract_mysql' AIRFLOW_CTX_EXECUTION_DATE='2025-09-18T04:29:04.296968+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-18T04:29:04.296968+00:00'
[2025-09-18T04:29:07.157+0000] {batch_ingestion_dag.py:29} INFO - Starting extraction from MySQL...
[2025-09-18T04:29:07.158+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-09-18T04:29:07.158+0000] {base.py:83} INFO - Using connection ID '***_db' for task execution.
[2025-09-18T04:29:07.245+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.9/site-packages/***/providers/common/sql/hooks/sql.py:215 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-09-18T04:29:07.278+0000] {batch_ingestion_dag.py:34} INFO - Extracted 5 records from MySQL and saved to /tmp/orders.csv.
[2025-09-18T04:29:07.278+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-09-18T04:29:07.287+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=batch_ingestion_dag, task_id=extract_mysql, execution_date=20250918T042904, start_date=20250918T042907, end_date=20250918T042907
[2025-09-18T04:29:07.336+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-09-18T04:29:07.357+0000] {taskinstance.py:3281} INFO - 1 downstream tasks scheduled from follow-on schedule check
